backend/app/middleware.py

```diff
@@ -66,66 +66,21 @@ def middleware(request):
 
     return middleware
 
-
-# https://stackoverflow.com/questions/2116860/django-session-cookie-domain-with-multiple-domains
-# https://ittone.ma/ittone/django-session_cookie_domain-with-multiple-domains/
-class SessionHostDomainMiddleware(SessionMiddleware):
```

Comment:
I remember this was created to deal with either the webview in the mobile app, or the tunnel. Please test both to make sure they are not broken now it is removed.

Reply:
Tunnel is working on the web and mobile app 👍